cmd/util.go

```diff
@@ -481,6 +500,33 @@ func getPlatformBenchmarkVersion(platform Platform) string {
 		}
 	case "vmware":
 		return "tkgi-1.2.53"
+	case "k3s":
+		switch platform.Version {
+		case "1.23":
+			return "k3s-cis-1.23"
+		case "1.24":
+			return "k3s-cis-1.24"
+		case "1.25", "1.26", "1.27":
+			return "k3s-cis-1.7"
+		}
+	case "rancher":
```

Comment:
I think it should be `rancher1`

Reply:
In our logic, we are always adding rancher1 to RKE gitVersion i.e, v1.26.8-rancher1.
Once it goes through getPlatformInfoFromVersion logic, it always yields "rancher"
You can check the example here - https://go.dev/play/p/x2IdtlQfov4